infra/MyStack.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Pulumi;
+using Pulumi.AzureNative.Resources;
+using Pulumi.AzureNative.Web;
+using Pulumi.AzureNative.Web.Inputs;
+
+// Define global variables
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        // Retrieve configuration and set config variables
+        var config = new Config(); 
+        var appServiceSkuName = config.Require("AppServiceSkuName");
+        var staticWebAppSkuName = config.Require("StaticWebAppSkuName");
+
+        // Create an Azure Resource Group
+        var resourceGroup = new ResourceGroup("rg-nebulift-dev-westeurope-");
```

Comment:
We should parameterize the ressource names and add it in the configuration to be able to deploy different environments like development and production (even if we will not need it here)